owncloudApp/src/main/res/layout/item_file_list.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout android:id="@+id/file_list_constraint_layout"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:minHeight="@dimen/item_file_list_min_height">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/icon_constraint_layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:focusable="false"
+        android:focusableInTouchMode="false"
+        android:minWidth="@dimen/item_file_list_icon_min_width"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageView
+            android:id="@+id/thumbnail"
+            android:layout_width="@dimen/file_icon_size"
+            android:layout_height="@dimen/file_icon_size"
+            android:src="@drawable/ic_menu_archive"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
+            android:id="@+id/localFileIndicator"
+            android:layout_width="@dimen/file_indicator_pin_size"
+            android:layout_height="@dimen/file_indicator_pin_size"
+            android:src="@drawable/sync_pin"
+            android:translationX="-8dp"
+            android:translationY="-10dp"
+            app:layout_constraintStart_toEndOf="@+id/thumbnail"
+            app:layout_constraintTop_toBottomOf="@+id/thumbnail" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/info_constraint_layout"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
```

Comment:
Instead of hardcoding it, you could create a new dimension value in [dims.xml](https://github.com/owncloud/android/blob/master/owncloudApp/src/main/res/values/dims.xml#L42), call it `standard_quarter_margin`  and set its value to 4dp

Reply:
Sure!